generators/server/templates/gradle.properties.ejs

```diff
@@ -30,6 +30,7 @@ yarn_version=<%= YARN_VERSION %>
 # Dependency versions
 jhipster_dependencies_version=<%= JHIPSTER_DEPENDENCIES_VERSION %>
 micronaut_version=<%=versions.micronaut%>
+micronautVersion=<%=versions.micronaut%>
```

Comment:
Should I deduplicate it? Micronaut function expects `micronautVersion` variable, but project already has `micronaut_version`. Duplication is minor, but I also don't want to break conventions in this project.

Reply:
Let's de-duplicate it I would say.